src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/features/form/components/HelpTextIcon.tsx

```diff
@@ -39,7 +39,7 @@ export default function HelpTextIcon(props: IHelpTextIconProps) {
         aria-hidden={false}
       >
         <i
-          className={`${classes.helpTextIcon} ${props.openPopover ? 'ai ai-circle-minus' : 'ai ai-circle-plus'}`}
+          className={`${classes.helpTextIcon} ${props.openPopover ? 'ai ai-circle-minus' : 'ai ai-help-popicon'}`}
```

Comment:
Just to be 100%, could you confirm this is the icon you had in mind @Febakke? The name does suggest so ;)

Reply:
The icon should be outline as default and filled when active, like this 
![image](https://user-images.githubusercontent.com/1464915/140728183-cd8e3a50-e58b-4ab3-80f0-f02742fe3843.png)

The icons we had was not perfectly overlapping and used different question marks, so I uploaded some new ones: 
"reg reg-help-outline"
"reg reg-help-filled"

I just have to find out how to include them in altinn-cdn.

Reply:
- Lastet opp nye ikoner til fortawesome. 
- Opprettet PR til altinn-cdn https://github.com/Altinn/altinn-cdn/pull/66
- Testet at det funker på lokal app
![image](https://user-images.githubusercontent.com/1464915/140933213-4ffd2b19-7527-46ee-82ea-f641f1cc2eef.png)

NB: Ikonet må få fargen #022F51  (blueDarker)